src/plugins/GUITestBase/src/GTUtilsDashboard.cpp

```diff
@@ -24,17 +24,78 @@
 #include <primitives/GTWidget.h>
 
 #include <QRegularExpression>
+#include <QStringBuilder>
 #include <QTabWidget>
 
 #include <U2Gui/HoverQLabel.h>
 
 #include "GTUtilsDashboard.h"
 
+namespace {
+
+// A substring with the name of the notification type is searched for. If the |str| contains exactly one type of
+// notification, then this type is returned, otherwise a null QVariant is returned.
+QVariant getNotificationTypeFromString(const QString &str) {
+    using Notification = U2::GTUtilsDashboard::Notification;
+
+    QVariant returnable;
+    constexpr int typesNumber = sizeof(Notification::typesNames) / sizeof(QString);
+    int counter = 0;
+    Notification::Type type;
+
+    for (int typeInd = 0; typeInd < typesNumber; ++typeInd) {
+        if (str.contains(Notification::typesNames[typeInd], Qt::CaseInsensitive)) {
```

Comment:
Use startsWith not contains. Reason: a user text may contain the same text as the notification type name.

Reply:
str is string like "<center><img src=":/U2Lang/images/error_20px.png"></center>", where a type is written in the png file name

Reply:
I see. In this case add 'class="warning|error|info"' marker to the notification HTML or use the existing class from the style-sheet (if this this available). 
In either case notification widget should have 100% info about notification type - no need to guess.